packages/back-end/src/services/vercel-native-integration.service.ts

```diff
@@ -0,0 +1,276 @@
+import { OrganizationInterface } from "back-end/types/organization";
+import { FeatureInterface } from "back-end/types/feature";
+import { ExperimentInterface } from "back-end/types/experiment";
+import { findVercelInstallationByOrganization } from "back-end/src/models/VercelNativeIntegrationModel";
+
+const VERCEL_URL = "https://api.vercel.com";
+
+const VERCEL_CLIENT_ID = process.env.VERCEL_CLIENT_ID || "";
+const VERCEL_CLIENT_SECRET = process.env.VERCEL_CLIENT_SECRET || "";
+
+export type VercelExperimentationItem = {
+  id: string;
+  slug: string;
+  origin: string;
+  category: "flag" | "experiment";
+  name?: string;
+  description?: string;
+  isArchived?: boolean;
+  createdAt?: number;
+  updatedAt?: number;
+};
+
+const FEATURE_ORIGIN = "app.growthbook.io";
```

Comment:
I think we have `APP_ORIGIN` as a constant already (set to `https://app.growthbook.io` on cloud)